packages/next/server/base-server.ts

```diff
@@ -1026,33 +1027,14 @@ export default abstract class Server<ServerOptions extends Options = Options> {
     const appPathRoutes: Record<string, string> = {}
 
     Object.keys(this.appPathsManifest || {}).forEach((entry) => {
+      if (entry.endsWith(NEXT_CLIENT_SSR_ENTRY_SUFFIX)) {
```

Comment:
The `__sc_client__` entries are included in the manifest for routing but it caused them to be routable here. Only reason I noticed was that `[...slug]` has a check for if it's the last segment or not and then throws.